zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -86,6 +86,7 @@
 
   /********************************** transient fields ******************************************/
   private transient boolean loaded = false;
+  private transient boolean executionAborted = false;
```

Comment:
oh, it's non-volatile object. It doesnt looks like thread safe code. 
You should use volatile w synchronized or something like java.util.concurrent.atomic.AtomicBoolean.

Reply:
i fixed it